ONSdigital/dp-hierarchy-builder
# Feature/fix audit job

### What
Updates:
- Go v1.15.8
- License 2021
- Dependencies

### How to review
- Check that `make test` passes and `make audit` shows no vulnerable dependencies
- Check the changes make sense

### Who can review

Anyone
